temi_code/curate_my_heart_counts_public_BRIDGE.R

```diff
@@ -181,33 +181,35 @@ curate_my_heart_counts <- function() {
     cardiovascular_satisfied_SchemaV2_v1,
     cardiovascular_satisfied_SchemaV3_v1,
     watchMotionActivityCollector_v1) %>%
-    as_tibble()
+    as_tibble() #changes dataframe into tibble format for cleaner look
   print("finished bind rows")
   return(my_heart_counts)
 }
 
-#
+#adds columns to mhc df which show length of engagement by Week and Day intervals
```

Comment:
The line by line comments you have below are excellent as we reverse engineer the code. as it gets more refined, more and more of those details should move to this function level comment

Reply:
sounds great! I will be looking to update them to more function level as we progress